core_lib/src/interface/scribblearea.cpp

```diff
@@ -522,12 +518,13 @@ void ScribbleArea::pointerMoveEvent(PointerEvent* event)
 
     if (event->buttons() & (Qt::LeftButton | Qt::RightButton))
     {
-        mOffset = getCurrentOffset();
 
         // --- use SHIFT + drag to resize WIDTH / use CTRL + drag to resize FEATHER ---
         if (currentTool()->isAdjusting())
         {
-            currentTool()->adjustCursor(mOffset.x(), event->modifiers()); //updates cursors given org width or feather and x
+            // TODO: make sure this doesn't break anything
+            // was mOffset.x()... but that should be independent and only accessible in selectino context
+            currentTool()->adjustCursor(currentTool()->getCurrentPoint().x(), event->modifiers()); //updates cursors given org width or feather and x
```

Comment:
`currentTool` already has access to it's own `getCurrentPoint`, and this is the only time the `adjustCursor` function is used, so we should just get rid of the first parameter.